cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/service/ServiceConfiguration.java

```diff
@@ -396,6 +393,23 @@ public Builder allowedNodes(String... allowedNodes) {
             return this.allowedNodes(Arrays.asList(allowedNodes));
         }
 
+        /**
+         * A list of all allowed nodes. CloudNet will choose the node with the most free resources.
+         * If a node is provided using {@link #node(String)}, this option will be ignored.
+         */
+        public Builder addAllowedNodes(Collection<String> allowedNodes) {
+            this.config.serviceId.allowedNodes.addAll(allowedNodes);
```

Comment:
The allowedNodes in the serviceId are null without any initialization, this will produce a NullPointerException.

Reply:
I have added a null check and if its null it just uses the set methods